catroid/src/main/java/org/catrobat/catroid/camera/CameraManager.java

```diff
@@ -543,18 +543,19 @@ public void destroyStage() {
 		}
 	}
 
-	public boolean switchToCameraWithFlash() {
+	public boolean isCameraFlashAvailable() {
+		return (frontCameraInformation.flashAvailable || backCameraInformation.flashAvailable);
```

Comment:
If you have a device with less than 2 cameras you run into a NPE here, because frontCameraInformation / backCameraInformation would not have been initialized in the CameraManager constructor. (Happened to me on an emulated device without camera)